docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
BTW, I tried to cover most if not all of the main issues discussed in this PR in the CLI/config roadmap:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/cli-roadmap.md

(though admittedly in less detail)